### DISCUSSION TITLE
`graph_from_adjacency_matrix` not defined in Julia

### DISCUSSION THREAD

**Discussion by ZhouHaobang** [oscar-system/Oscar.jl#3049] [Q&A]
My code:

julia> G = ZZ[0 0; 1 0]
julia> graph_from_adjacency_matrix(Directed, G)

ERROR: UndefVarError: `graph_from_adjacency_matrix` not defined
Stacktrace:
 [1] top-level scope
   @ REPL[61]:1

**Comment by simonbrandhorst** [comment]
It works for me. 
`graph_from_adjacency_matrix` is an Oscar method. Did you load Oscar?
```
using Oscar
```
